source/fem-dem/vans_assemblers.cc

```diff
@@ -1310,8 +1306,6 @@ GLSVansAssemblerGidaspow<dim>::calculate_particle_fluid_interactions(
 
       particle_number += 1;
     }
-
-  beta_drag = beta_drag / scratch_data.cell_volume;
```

Comment:
I'm asking myself the same question here.

Reply:
The term of the beta drag in all the drag models should be 1/s to be equal to the units in the VANS equation. Before the momentum_transfer_coefficient unit was m^3/s so we had to divide by the element volume to get 1/s. In the Gidaspow model, the momentum_transfer_coefficient unit is already 1/s . So dividing beta by the element volume would result in the beta unit to be 1/(m^3.s) and this was leading to a very very small initial bubble in the fluidized bed. It is just a matter of units

Reply:
Ohh I see, so the original correlation does not lead to the same unit by definition? Interesting. It's true that the unit of the Navier-Stokes equation we solve are Velocity / time so L^1 / T^2. Since beta is multiplied by the velocity, it needs to have units of seconds inverse.